src/Fixer/Comment/NoEmptyCommentFixer.php

```diff
@@ -60,7 +60,11 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 continue;
             }
 
-            [$blockStart, $index, $isEmpty] = $this->getCommentBlock($tokens, $index);
+            $blockInfo = $this->getCommentBlock($tokens, $index);
+            $blockStart = $blockInfo['blockStart'];
+            $index = $blockInfo['blockEnd'];
+            $isEmpty = $blockInfo['isEmpty'];
```

Comment:
```suggestion
            extract($this->getCommentBlock($tokens, $index));
```
JK 😆

Reply:
It is very good idea for me - let me handle it!
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7684